app/Services/UserService.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App\Services;
+
+class UserService extends Service {
+    
+    public function createUser($data = array()) {
```

Comment:
Service methods should operate with object, so instead of '$data = array()' method is supposed to accept user model 'User $user = null'